keadm/cmd/keadm/app/cmd/util/edgecoreinstaller.go

```diff
@@ -93,6 +93,11 @@ func (ku *KubeEdgeInstTool) createEdgeConfigFiles() error {
 
 	edgeCoreConfig := v1alpha2.NewDefaultEdgeCoreConfig()
 	edgeCoreConfig.Modules.EdgeHub.WebSocket.Server = ku.CloudCoreIP
+	host, _, err := net.SplitHostPort(ku.CloudCoreIP)
+	if err != nil {
+		return fmt.Errorf("get current host and port failed: %v", err)
+	}
+	edgeCoreConfig.Modules.EdgeHub.Quic.Server = net.JoinHostPort(host, strconv.Itoa(constants.DefaultQuicPort))
```

Comment:
add `edgeCoreConfig.Modules.EdgeHub.Quic.Server = net.JoinHostPort(cloudCoreIP, strconv.Itoa(constants.DefaultQuicPort))` at line 127 directly is better. 
![image](https://github.com/kubeedge/kubeedge/assets/48788150/29273255-1080-406d-a882-1ef2377e4425)

Reply:
ok ，I have made it.